Source/ImageGlass/LocalSetting.cs

```diff
@@ -87,6 +87,12 @@ public static class LocalSetting
         /// </summary>
         public static string InitialInputImageFilename { get; set; } = "";
 
+        /// <summary>
+        /// The current image sorting order. This is a temporary override which takes place when respecting
+        /// the user's sorting order from Windows Explorer.
+        /// </summary>
+        public static ImageOrderBy ImageLoadingOrder { get; set; }
+        public static bool ImageLoadingDescending { get; set; }
```

Comment:
Please add description for this property and rename to `IsImageOrderDesc`. Why don't we just use `GlobalSetting.IsImageOrderDesc`?